platform/linux-generic/include/odp_ipsec_internal.h

```diff
@@ -158,6 +160,8 @@ struct ipsec_sa_s {
 	void		*context;
 	odp_queue_t	queue;
 
+	uint32_t	combined_mode;
+	uint32_t	insert_seq_hi;
```

Comment:
Since these are just simple flags, there is no need to reserve a full uint32_t for each of them, especially since there is an unnamed structure of single bit flags just a little bit below this. You could put these in there as single bit flags, just like you did for the esn flag.

Reply:
Ack, will move.